tests/python/relay/test_op_level3.py

```diff
@@ -87,6 +87,13 @@ def test_transpose_infer_type():
     assert yy.checked_type == relay.TensorType(
         (t, n, 100), "float32")
 
+    y = relay.transpose(x)
+    "axes=" in y.astext()
```

Comment:
What does this line mean?

Reply:
I believe he is checking that the text formatting code correctly adds the axes argument.

Reply:
Should we add an assert here?